lib/feed.xml

```diff
@@ -74,6 +74,11 @@
       {% if post.category %}
         <category term="{{ post.category | xml_escape }}" />
       {% endif %}
+      {% if post.categories %}
+        {% for category in post.categories %}
+        <category term="{{ category | first | xml_escape }}" />
```

Comment:
I don't think we need `first`, as we're only working with one category at a time.